generic_pla.xml.fdm_material

```diff
@@ -27,7 +27,7 @@ Generic PLA profile. The data in this file may not be correct for your specific
         <setting key="break preparation temperature">200</setting>
         <setting key="break position">-50</setting>
         <setting key="break speed">25</setting>
-        <setting key="break temperature">50</setting>
+        <setting key="break temperature">65</setting>
```

Comment:
This probably doesn't belong here.

Reply:
@robinmdh

Reply:
Updated to 55 per the parameter specification.